graphitesend/graphitesend.py

```diff
@@ -114,7 +114,10 @@ def __init__(self, prefix=None, graphite_server=None, graphite_port=2003,
         self.lastmessage = None
 
         self.lowercase_metric_names = lowercase_metric_names
-        self.asynchronous = asynchronous
+        if asynchronous:
```

Comment:
Lets add this from @Lujeni , fix up the merge conflicts and merge in this sucker :)